driver/normalizer/parser.go

```diff
@@ -25,9 +25,25 @@ var ToNoder = &native.ObjectToNoder{
 	//      children node properties.
 }
 
+// transUASTParserBuilder creates a parser that transform source code files into *uast.Node
```

Comment:
oh, I meant to remove the function name from the docstring, not the whole docstring, I didn't express it correctly. Anyway I think it's ok this way too.